gradle/shipkit.gradle

```diff
@@ -5,7 +5,7 @@ shipkit {
     team.developers = ['mockitoguy:Szczepan Faber', 'mstachniuk:Marcin Stachniuk',
                        'wwilk:Wojtek Wilk', 'epeee:Erhard Pointl', 'NagRock:Maciej Kuster']
 
-    releaseNotes.publicationRepository = "https://plugins.gradle.org/plugin/org.shipkit.java"
+    releaseNotes.publicationRepository = "https://plugins.gradle.org/plugin/org.shipkit.java/"
```

Comment:
What if we make the getter add a slash at the end if it doesn't end with slash yet? This way user doesn't have to care about it. Maybe we can encapsulate this logic in a utility method and use it for all getters that return URLs and are part of the public API?
@mockitoguy What do you think?

Reply:
@wwilk This would break the powermock use case, isn't it?
Currently, we are using
`releaseNotes.publicationRepository = "https://bintray.com/powermock/maven/powermock/powermock-"`
in powermock in order to get links like https://bintray.com/powermock/maven/powermock/powermock-1.7.3

Reply:
True, good catch! But anyway, we should stick to one convention for all default values that we provide. So maybe all of them should have slash at the end?